ensenso_camera/include/ensenso_camera/stereo_camera.h

```diff
@@ -36,6 +36,7 @@ class StereoCamera : public Camera
   sensor_msgs::CameraInfoPtr rightCameraInfo;
   sensor_msgs::CameraInfoPtr leftRectifiedCameraInfo;
   sensor_msgs::CameraInfoPtr rightRectifiedCameraInfo;
+  sensor_msgs::CameraInfoPtr depthImageCameraInfo;
```

Comment:
Nicht nötig, die hier global zu speichern. Ich würde in `onRequestData` eine Kopie anlegen mit `auto depthImageCameraInfo = boost::make_shared<sensor_msgs::CameraInfo>(*leftRectifiedCameraInfo)` und darauf dann den Offset anwenden.